src/OrchardCore.Modules/OrchardCore.Localization/Services/TranslationsManager.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using OrchardCore.Localization.Models;
+
+namespace OrchardCore.Localization.Services
+{
+    public class TranslationsManager : ITranslationsManager
+    {
+        public async Task<TranslationsDocument> GetTranslationsDocumentAsync()
+        {
+            // TODO: Fetch the translations from the database
+            var document = new TranslationsDocument();
+            await Task.CompletedTask;
+
+            return document;
+        }
+        
+        public async Task UpdateTranslationAsync(string name, Translation template)
+        {
+            // TODO: update the translations into the database
+            await Task.CompletedTask;
+        }
```

Comment:
So, will need to be redone if intended to be stored and kept in sync in our multi level cache, e.g to be shared and kept in sync across multiple running instances, see the **`TemplatesManager`** as an example